dags/bqetl_search_terms_daily.py

```diff
@@ -91,6 +91,24 @@
         arguments=["--schema_update_option=ALLOW_FIELD_ADDITION"],
     )
 
+    search_terms_derived__search_terms_daily__v1 = bigquery_etl_query(
```

Comment:
See PR comment in `query.sql`. I don't think this can land as-is as airflow access to unsanitized search terms is very limited.